roles/robottelo/defaults/main.yml

```diff
@@ -3,5 +3,27 @@ robottelo_url: https://github.com/SatelliteQE/robottelo.git
 robottelo_directory: /root/robottelo
 robottelo_virtualenv_path: /root/robottelo/venv
 robottelo_version: master
-robottelo_test_arguments:
-  - tests/foreman/endtoend/test_api_endtoend.py
+robottelo_junit_file: results.xml
+robottelo_test_type: endtoend # one of [tier1|tier2|tier3|destructive|upgrade|endtoend]
+robottelo_test_endpoints: # [api|ui|cli]
+  - api
+robottelo_test_options: >
+  {% if robottelo_test_type == 'endtoend' %}
+    not stubbed
+  {% else %}
+    not stubbed and {{ robottelo_test_type }}
+  {% endif %}
+robottelo_test_path: >
+  {% if robottelo_test_type == 'destructive' %}
+    tests/foreman/sys
+  {% else %}
+    {% if robottelo_test_type == 'endtoend' %}
```

Comment:
I thought there was a `elif` in Jinja templates which might be a bit more readable.